release_notes.md

```diff
@@ -1,118 +1,57 @@
 ## What's New
 
 >[!NOTE]
->This solution pack requires FortiSOAR `v7.6.0` and later.
+>This solution pack requires FortiSOAR `v7.6.1` and later.
 
-### Outbreak Management - A New Look!
+### Enhanced Performance of Threat Intel Feed Ingestion
 
-Get ready to experience a revamped Outbreak Management, designed to streamline your investigation workflow and enhance your visibility of your environment to respond and remediate these Outbreak Alerts effectively and efficiently.
+Experience a significant boost in performance during the ingestion of Threat Intel Feeds for Outbreak Alerts:
 
-When you click Outbreak Management in the navigation menu, you’re greeted with an intuitive layout featuring these powerful tabs:
+**Bulk Ingestion**: The individual IOC creation process in playbooks has been replaced with a Bulk Ingestion step. This enhancement leverages a connector action to trigger a bulk ingestion playbook that batches IOCs into groups and processes them asynchronously. This approach ensures faster and more efficient processing of threat intelligence feeds.
 
-- **Dashboard**: Stay informed with a dynamic Outbreak Response Overview, showcasing:
-    - Outbreak Status (Last 30 Days)
-    - KEVs For Outbreak CVEs (Last 30 Days)
-    - Recent Outbreaks Detected
-    - Top 10 Outbreak Threat Feeds
-    - …and so much more!
+### Automated Updates for Outbreak Alerts
 
-- **Outbreaks**: Access comprehensive records of all outbreak alerts at your fingertips.
+A new feature automatically updates critical information for Outbreak Alerts with the statuses *New* or *Tracking*. This is achieved using the **Update Outbreak Alert Details** playbook, which updates key details such as CVEs, background information, and descriptions. The playbook is scheduled to run daily, ensuring that alerts remain current and accurate.
```

Comment:
This needs to be added in usage.md as well with a screenshot of the schedule and outbreak alert details page before and after of this playbook

Reply:
please also mention that user can disable this feature by deactivating the said schedule

Reply:
Added a note in usage.